src/builder/build_traits.rs

```diff
@@ -1,3 +1,4 @@
+#![allow(missing_docs)]
```

Comment:
🤨

Reply:
Yes, this is not for submission, but good to flag up, sorry. If we decide to go this way then I'll take this out and put in docs, but at least it shows that the rest of CI passes